base/build.sh

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
I call this directory `runtime-image`, based on the suggestion of the howto.  It would be nice if we were consistent, but I don't particularly have a preference.